### ISSUE TITLE
Update to Sixense Core API

### ISSUE THREAD

**Issue by mrwonko** [mrwonko/PySixense#3]
> Just came out today.
> 
> You don't have to do anything with it beyond changing the DLLs and the pysixense.pyd works great!
> 
> I added the new
> sixense_x64.dll
> and removed all the other dlls that were previously being used and it works.
> [..]
> Paul

So yeah, use that.

**Comment by yabix007** [comment]
Is it possible to import pysixense in python. How to install the Library?

**Comment by mrwonko** [comment]
The latest release can be found [here](https://mrwonko.de/downloads/2015/pysixense-34.html), it's for Python 3.4 and almost 10 years old. Once unpacked, you can import the PySixense.pyd file, as [outlined in the documentation](https://github.com/mrwonko/PySixense/blob/master/doc.markdown).

**Comment by yabix007** [comment]
Does it work with Python 3.13 or 3.8?

**Comment by mrwonko** [comment]
No, it probably only works for Python 3.4. If you download the source code from here and install Boost, you might be able to compile a version for the latest Python, if there were no changes to the API in the meantime.

**Comment by yabix007** [comment]
Thank you for your effort. Iam using FreePie instead. I think there are too many changes made in the API.
